data/selectors.json

```diff
@@ -1,5 +1,5 @@
 {
-  "label": ".ant-form-item-label",
+  "label": ".ant-form-item-required",
```

Comment:
Deleting label ".ant-form-item-label" crashes other people's tests.
Please return the line         "label": ".ant-form-item-label"
You can use the original selector for your test OR you can add your own selector, but please do not delete existing code from other students.

Reply:
If I can change it back, then my tests don't work. I tried in console your $$('ant-form-item-label') and it returned nothing. Julia recommended to use  ".ant-form-item-required" as an array, so [2] would be age field. Please advise.

Reply:
Check $$('ant-form-item-label') in the console of the developer's tools and you'll see that it picks up all 5 fields, while yours picks up only 4. It was discussed on group meetings and on Julia's class that it's better to use $$('ant-form-item-label').
Hint: to make your test to go through try to use a different command. getText() instead of getAttribute('title') will do the job.

Reply:
Updated. Commit2